portal-frontend/src/app/features/review-application/review-submit-fng/review-submit-fng.component.ts

```diff
@@ -38,7 +39,14 @@ export class ReviewSubmitFngComponent implements OnInit, OnDestroy {
     private applicationDocumentService: ApplicationDocumentService
   ) {}
 
+  @HostListener('window:resize', ['$event'])
+  onWindowResize() {
+    this.isMobile = window.innerWidth < 480;
```

Comment:
I think it is worth moving breakpoint values to constants since we are using it in multiple places

Reply:
Done!